onmt/trainer.py

```diff
@@ -269,14 +269,18 @@ def train(self,
                 step, train_steps,
                 self.optim.learning_rate(),
                 report_stats)
+            if step % self.report_manager.report_every == 0:
```

Comment:
I am not sure this is what we want.
My understanding is that:
report_stats reports stats between two report_every
total_stats cumulates for the training.
if we do what you suggest, I have the feeling that report_stats is no longer needed.